README.md

```diff
@@ -0,0 +1,157 @@
+| `NEOFS_HTTP_GATE`      | HTTP Gateway domain address                                                           | **No**   | http.fs.neo.org        |
+| `STORE_OBJECTS_CID`    | Container ID for your data. For example: 7gHG4HB3BrpFcH9BN3KMZg6hEETx4mFP71nEoNXHFqrv | **Yes**  | N/A                    |
+
+
+### Workflow environment variables
+The following variables must be passed as
+[GitHub Actions vars context](https://docs.github.com/en/actions/learn-github-actions/variables#using-the-vars-context-to-access-configuration-variable-values)
+or [GitHub Actions environment variables](https://docs.github.com/en/actions/learn-github-actions/variables).
+
+| Key                 | Value                                                                                                                                                   | Required | Default |
+|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------|
+| `PATH_TO_FILES_DIR` | Path to the directory with the files to be pushed                                                                                                       | **Yes**  | N/A     |
+| `NEOFS_ATTRIBUTES`  | User attributes in form of Key1=Value1,Key2=Value2. By default, each object contains attributes of relative path to the file and MIME type of the file. | **No**   | N/A     |
+| `URL_PREFIX`        | Prefix to the url address for each of the files(objects)                                                                                                | **No**   | N/A     |
+
+### Expiration period environment variables
+The following variables must be passed as 
+[GitHub Actions vars context](https://docs.github.com/en/actions/learn-github-actions/variables#using-the-vars-context-to-access-configuration-variable-values)
+or [GitHub Actions environment variables](https://docs.github.com/en/actions/learn-github-actions/variables).
+
+These environment variables are responsible for the storage time of the results in the storage network in epochs 
+(in the mainnet, an epoch is approximately equal to one hour, so we can assume that values are specified in HOURS).
+
+After the period is over, the data will be deleted. They are convenient to use for log rotation or test reports.
+
+They default to 0, in which case the data will be stored until they are manually deleted.
+We recommend setting a reasonable and convenient for work expiration period, for example, a month (744 hours).
+
+
+| Key                 | Value                                    | Required | Default |
+|---------------------|------------------------------------------|----------|---------|
+| `EXPIRATION_PERIOD` | Expiration period for uploaded artifacts | **No**   | 0       |
```

Comment:
LIFETIME? I think that's the way it's named in CLI.

Reply:
It's not critical to use `--lifetime` (we upload a set of objects and they can get different expirations this way), but it'd just be a better name, `period` implies some repetitiveness which is not the case here.

Reply:
Fixed